azure-pipelines/Format-LocalizationFiles.ps1

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env pwsh
+
+[CmdletBinding()]
+$Root = Resolve-Path -LiteralPath "$PSScriptRoot/.."
+
+Get-ChildItem "$Root/locales/messages.*.json" | ? {
+  (Split-Path -Leaf $_) -ne 'messages.en.json'
+} | % {
+    $fileName = $_
+    Write-Host "Formatting $fileName"
+    $fileContents = Get-Content $fileName
+    $fileContents = $fileContents -join "`n" # use LF, not CRLF
+    if (-not $fileContents.EndsWith("`n"))
+    {
+        $fileContents += "`n" # add a trailing newline
+    }
+    # this (convert to UTF-8 followed by WriteAllBytes) avoids adding a BOM in Windows PowerShell
+    $fileContents = [System.Text.Encoding]::UTF8.GetBytes($fileContents)
+    [io.file]::WriteAllBytes($fileName, $fileContents)
```

Comment:
```suggestion
    # avoid adding a BOM
    $encoding = New-Object -TypeName System.Text.UTF8Encoding -ArgumentList @($false, $true)
    [System.IO]::WriteAllText($fileName, $fileContents, $encoding)
```

This lets the encoding happen with FileStream's preferred buffer sizes rather than needing to encode the thing as a whole chunk.

I know, I know, perf of this doesn't matter, so feel free to ignore this but smashing the whole file into another buffer again hurts inside 😅

Reply:
(Also, false/true is no BOM yes fail on encoding errors: https://docs.microsoft.com/en-us/dotnet/api/system.text.utf8encoding.-ctor?view=net-6.0#system-text-utf8encoding-ctor )

Reply:
I am ignoring this as it doesn't matter, but noted! Thanks Billy!